src/elm/Page/Dashboard/Analysis.elm

```diff
@@ -595,15 +636,13 @@ fetchAnalysis { selectedCommunity, shared, authToken } { profile, statusFilter }
         filterRecord =
             { claimer = optionalClaimer
             , status = optionalStatus
-            }
-
-        filter =
-            case ( filterRecord.claimer, filterRecord.status ) of
-                ( Absent, Absent ) ->
-                    Absent
+            , direction =
+                if direction == ASC then
+                    Present Cambiatus.Enum.Direction.Asc
 
-                ( _, _ ) ->
-                    Present filterRecord
+                else
+                    Present Cambiatus.Enum.Direction.Desc
```

Comment:
Using a `case` statement here would be more appropriate (personally, it always seems kind of wrong to compare type constructors like this). For now, we're only sorting ascending and descending, but if someday we decide to sort by pending/accepted for example, it would be easier to know where to change it, and we would avoid possible weird bugs

Reply:
nice! I'll change it!